pkgs/development/libraries/openssl/default.nix

```diff
@@ -221,8 +221,6 @@ in {
       (if stdenv.hostPlatform.isDarwin
        then ./use-etc-ssl-certs-darwin.patch
        else ./use-etc-ssl-certs.patch)
-    ] ++ lib.optionals (stdenv.isDarwin && (builtins.substring 5 5 version) < "m") [
-      ./1.1/macos-yosemite-compat.patch
```

Comment:
Nice! With this change, I believe the `rec` a few lines above can also be removed.